x-pack/plugins/security_solution/common/detection_engine/schemas/request/rule_schemas.test.ts

```diff
@@ -656,24 +421,20 @@ describe('create rules schema', () => {
   });
 
   test('saved_query type can have filters with it', () => {
-    const payload: CreateRulesSchema = {
-      ...getCreateRulesSchemaMock(),
+    const payload: SavedQueryCreateSchema = {
```

Comment:
👍 on keeping the type here. With a spread and an additional value, we have to explicitly add the type to ensure stricter checks from TS. Awesome!